src/frontend/magnify/src/components/profile/IdentityForm/IdentityForm.test.tsx

```diff
@@ -21,7 +22,7 @@ describe('IdentityForm', () => {
     await user.clear(nameInput);
     fireEvent.blur(nameInput);
 
-    await screen.findByText('name is a required field');
+    await screen.findByText(validationMessages.required.defaultMessage);
```

Comment:
In this case, we prefer to use a static value. First it helps to better understand the context of this test. Then it warns us about regression related to the label.